src/PublisherInterface.ts

```diff
@@ -260,14 +262,19 @@ export class PublisherInterface {
    * publisherInterface.addListener("FrameMoved", (targetId)=>{console.log(targetId)}));
    * ```
    * @param eventName - A case-sensitive string representing the editor event type to listen for.
-   * @param callbackFunction - A function that executes when the event is triggered.
+   * @param callbackFunction - A function that executes when the event is triggered. If callback is null, the listener will instead call window.OnEditorEvent
    */
   public async addListener(
     eventName: string,
-    callbackFunction: (targetId: string) => void
+    callbackFunction?: (targetId: string) => void
   ): Promise<void> {
-    !this.chiliEventListenerCallbacks.has(eventName) &&
-    this.chiliEventListenerCallbacks.set(eventName, callbackFunction);
+
+    console.log(this);
```

Comment:
```suggestion
```

Reply:
@pietervp LOL, I even made an conscience effort to remove this. Like I saw it, and I swear I deleted it,  but I guess I did not. Good catch.